src-docs/src/views/toast/toast_list.js

```diff
@@ -72,6 +72,7 @@ export default () => {
               Please move your mouse to show that you&rsquo;re still using
               Kibana.
             </p>
+            <EuiButton autoFocus>Extend my session</EuiButton>
```

Comment:
I'm experiencing REALLY weird behavior with this toast on Firefox. Whenever this toast and I click away, ANYWHERE on the page (even to devtools - was trying to log `document.activeElement`), the entire toast disappears. Are you seeing that behavior as well?

Also, separately, should we exemplify the return focus behavior by doing something like

```suggestion
            <EuiButton autoFocus onClick={() => document.getElementById('addToastButton').focus()}>Extend my session</EuiButton>
```
(We'd need to said a static ID to the add toast button)

Reply:
I'm experiencing this in Chrome too. Tabbing forward away from the group of toast messages causes it to disappear, but just the one with the autofocused button. Going to experiment a bit more and see if it's the `autoFocus` causing it.

Reply:
ah fuckin jeez lol 😐 💀 It looks EuiGlobalToastList _might_ be causing it??

https://github.com/elastic/eui/blob/cb0db20b63bb9f17e9bee514e667d6df4cbbe0ed/src/components/toast/global_toast_list.tsx#L276-L279

I